.github/workflows/ci-build.yml

```diff
@@ -33,5 +33,17 @@ jobs:
           cache: yarn
       - name: Install dependencies
         run: yarn install --frozen-lockfile
+      - name: Set workflow variables
+        # Id is needed to access output in a next step.
+        id: vars
+        run: |
+          echo "SHORT_SHA=$(git rev-parse --short HEAD)" >> "$GITHUB_OUTPUT"
       - name: Build app
         run: yarn build
+      - name: Upload build artifacts
+        # Upload build artifacts on push event.
+        if: github.event_name == 'push'
+        uses: actions/upload-artifact@v3
+        with:
+          name: oasis-wallet-web-${{ steps.vars.outputs.SHORT_SHA }}
```

Comment:
Does it make sense to omit the `-web` part or will we build different artifacts for different form factors (i.e. web, browser extension and mobile app)?

Reply:
For now I will remove `-web` suffix. Long term it depends if we want to have extension build artifacts too (not sure about mobile)

Reply:
According to @lubej we can have mobile build artifacts too, but an issue might be signing with certificate.

Reply:
Ok, in this case, let's add the `-web` suffix.